File: JS-Basics/For Loop/Exercise/03.js
function calculateHistogram(input) {
    const n = input[0]; 
    const numbers = input.slice(1); 

    const p1 = (numbers.filter(num => num < 200).length / n * 100).toFixed(2);
    const p2 = (numbers.filter(num => num >= 200 && num < 400).length / n * 100).toFixed(2);
    const p3 = (numbers.filter(num => num >= 400 && num < 600).length / n * 100).toFixed(2);
    const p4 = (numbers.filter(num => num >= 600 && num < 800).length / n * 100).toFixed(2);
    const p5 = (numbers.filter(num => num >= 800).length / n * 100).toFixed(2);

    console.log(`${p1}%`);
    console.log(`${p2}%`);
    console.log(`${p3}%`);
    console.log(`${p4}%`);
    console.log(`${p5}%`);
}

File: Basics/For Loop/Exercise/03.py
num = int(input())

p1 = 0
p2 = 0
p3 = 0
p4 = 0
p5 = 0

for i in range(1, num + 1):
    number = int(input())
    if number < 200:
        p1 += 1
    elif 200 <= number <= 399:
        p2 += 1
    elif 400 <= number <= 599:
        p3 += 1
    elif 600 <= number <= 799:
        p4 += 1
    elif number >= 800:
        p5 += 1

p1 = p1 / num * 100
p2 = p2 / num * 100
p3 = p3 / num * 100
p4 = p4 / num * 100
p5 = p5 / num * 100

print(f"{p1:.2f}%")
print(f"{p2:.2f}%")
print(f"{p3:.2f}%")
print(f"{p4:.2f}%")
print(f"{p5:.2f}%")
